README.rst

```diff
@@ -22,7 +22,17 @@ Define installation steps here.
 From source
 ~~~~~~~~~~~
 
-Define installation steps here.
+1)  sudo -u pulp -i
+2)  source ~/pulpvenv/bin/activate
+3)  git clone https://github.com/pulp/pulp\_ansible.git
+4)  cd pulp\_ansible
```

Comment:
pulp_ansible   ?

Reply:
The slash is to escape the underscore. See:

https://github.com/pulp/pulp_file/blame/master/README.rst#L44

Reply:
thanks